src/server/sched_func.c

```diff
@@ -981,10 +957,21 @@ set_sched_default(pbs_sched *psched, int from_scheduler)
 		psched->sch_attr[SCHED_ATR_throughput_mode].at_flags |= ATR_SET_MOD_MCACHE | ATR_VFLAG_DEFLT;
 	}
 
+	if (!(psched->sch_attr[SCHED_ATR_version].at_flags & ATR_VFLAG_SET)) {
+		set_attr_generic(&(psched->sch_attr[SCHED_ATR_version]), &sched_attr_def[SCHED_ATR_version], PBS_VERSION, NULL, SET);
+		psched->sch_attr[SCHED_ATR_version].at_flags |= ATR_SET_MOD_MCACHE | ATR_VFLAG_DEFLT;
```

Comment:
If you do keep this, no need to set ATR_SET_MOD_MCACHE.  set_attr_generic() sets that.